delainewendling/budgett
# Side Nav

### Pr Description

This PR adds a side navbar with the appropriate logos (not the appropriate CSS)

### Steps to Test

1. Go into the Client folder and run `webpack-dev-server --inline --hot`
1. Make sure a navbar comes up with the logo and three icons on it
